lib_eio_posix/flow.ml

```diff
@@ -71,7 +71,11 @@ let of_fd fd = object (_ : <Eio_unix.Net.stream_socket; Eio.File.rw>)
   method read_methods = []
   method copy src = copy src fd
 
-  method pread ~file_offset bufs = Low_level.preadv ~file_offset fd (Array.of_list bufs)
+  method pread ~file_offset bufs =
+    let got = Low_level.preadv ~file_offset fd (Array.of_list bufs) in
```

Comment:
The array/list mismatch in the low-level/high-level interface leads to more allocations, but I can't see how to get rid of that easily given what the underlying syscall expects.

Reply:
I know I wouldn't mind the high-level interface exposing array. That way I can choose whether to convert my list to an array or adapt my algorithm to use array.